utils/rpcd-mod-poe/Makefile

```diff
@@ -0,0 +1,40 @@
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=rpcd-mod-poe
+PKG_VERSION:=0.1
+PKG_RELEASE:=1
+PKG_LICENSE:=GPL-2.0
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/rpcd-mod-poe
+  SECTION:=utils
+  CATEGORY:=Base system
+  TITLE:=ubus poe interface
+  MAINTAINER:=Paul Spooren <mail@aparcar.org>
+  DEPENDS:=rpcd kmod-i2c-gpio-custom
```

Comment:
Please use selecting dependencies `DEPENDS:=+rpcd +kmod-i2c-gpio-custom`

Reply:
I suggest to captialize the title and use proper casing for PoE as well, e.g. `TITLE:=OpenWrt ubus PoE interface`